src/rp2_common/pico_cyw43_driver/cyw43_bus_pio_spi.c

```diff
@@ -84,6 +84,16 @@ typedef struct {
 
 static bus_data_t bus_data_instance;
 
+int cyw43_spi_clock_div_int = CLOCK_DIV;
```

Comment:
coudn't these be static const?

Reply:
They are set by cyw43_spi_set_clock_divisor() so they can't be const, but they should be static.  I've made the changes in the latest commit.